vagrant_scripts/configure_pyenv.sh

```diff
@@ -0,0 +1,11 @@
+echo -e 'if shopt -q login_shell; then' \
+  '\n  export PYENV_ROOT="$HOME/.pyenv"' \
+  '\n  export PATH="$PYENV_ROOT/bin:$PATH"' \
+  '\n eval "$(pyenv init --path)"' \
+  '\nfi' >> $HOME/.bashrc
+echo -e 'if [ -z "$BASH_VERSION" ]; then'\
+  '\n  export PYENV_ROOT="$HOME/.pyenv"'\
+  '\n  export PATH="$PYENV_ROOT/bin:$PATH"'\
+  '\n  eval "$(pyenv init --path)"'\
+  '\nfi' >> $HOME/.profile
+source $HOME/.profile
```

Comment:
You don't need this line if this file is only to be used as a vagrant script (as the folder name implies). The .profile will run when it starts a shell for the next script.